### PR TITLE
MEN-2173 More descriptive information for the user when conversion is complete

### PR DESCRIPTION
**Pull request by DominikAdamski**
Added information where user can find output files for give stage of conversion.

MEN-2173

Signed-off-by: Dominik Adamski <adamski.dominik@gmail.com>

### CODE REVIEW

File: mender-convert [hunk 1]
```diff
@@ -723,27 +723,41 @@ sudo true
 case "$1" in
   raw-disk-image-shrink-rootfs)
     do_raw_disk_image_shrink_rootfs || rc=$?
-  [[ $rc -ne 0 ]] && { return 1; }
+    [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
```
Comment (estenberg) [inline mender-convert]: The rootfs partition in the raw disk image has been shrinked successfully!
You can now convert the output raw disk image ($raw_disk_image) to a Mender disk image.

File: mender-convert [hunk 2]
```diff
@@ -723,27 +723,41 @@ sudo true
...
+    echo "Shrinked Linux image you can convert to Mender disk image is at "$raw_disk_image
     ;;
   raw-disk-image-create-partitions)
     do_raw_disk_image_create_partitions || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
```
Comment (estenberg) [inline mender-convert]: A new Mender disk image with partition layout to support Mender has been successfully created!
You can find the output Mender disk image at $mender_disk_image.

File: mender-convert [hunk 3]
```diff
@@ -723,27 +723,41 @@ sudo true
...
+    echo "The disk image with modified partition table you can find at "$mender_disk_image
     ;;
   install-mender-to-mender-disk-image)
     do_install_mender_to_mender_disk_image || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
```
Comment (estenberg) [inline mender-convert]: The Mender client has been successfully installed to the Mender disk image.
You can find the output Mender disk image at $mender_disk_image.

File: mender-convert [hunk 4]
```diff
@@ -723,27 +723,41 @@ sudo true
...
+    echo "The disk image with configured Mender client you can find at "$mender_disk_image
     ;;
   install-bootloader-to-mender-disk-image)
     do_install_bootloader_to_mender_disk_image || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
```
Comment (estenberg) [inline mender-convert]: A bootloader configuration supporting dual A/B rootfs updates has been installed to the Mender disk image!
You can find the output Mender disk image at $mender_disk_image.

File: mender-convert [hunk 5]
```diff
@@ -723,27 +723,41 @@ sudo true
...
+    echo "The disk image with installed bootloader for Mender updates you can find at "$mender_disk_image
     ;;
   mender-disk-image-to-artifact)
     do_mender_disk_image_to_artifact || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Stage complete!"
```
Comment (estenberg) [inline mender-convert]: A Mender Artifact has been successfully extracted from your Mender disk image!
You can find the Mender Artifact at $mender_artifact and use it to deploy updates.

File: mender-convert [hunk 6]
```diff
@@ -723,27 +723,41 @@ sudo true
...
+    echo "The Mender Artifact you can upload to your Mender server to deploy to your devices is at "$mender_artifact
     ;;
   from-raw-disk-image)
     do_from_raw_disk_image || rc=$?
     [[ $rc -ne 0 ]] && { exit 1; }
+    echo "Conversion complete!"
+    echo "The Mender root file system partition is at "$mender_rootfs_image
+    echo "The Mender image you can provision your device storage with is at "$mender_disk_image
```
Comment (estenberg) [inline mender-convert]: The Mender disk image you can provision your device storage with is at "$mender_disk_image.

Comment (estenberg) [reply mender-convert]: Would move this line first, then the Mender root file system partition in the middle (first Mender disk image, then Mender root file system partition, then Mender Artifact).

**Comment by apodogrocki** [comment]
@estenberg I have push the commit with changes requested by you. Please check if it is OK now.

**Comment by estenberg** [comment]
:+1:

**Pull request merged**